web/packages/teleport/src/config.ts

```diff
@@ -34,15 +34,15 @@ import type {
 
 import type { SortType } from 'teleport/services/agents';
 import type { RecordingType } from 'teleport/services/recordings';
+import type { ParticipantMode } from 'teleport/services/session';
 import type { WebauthnAssertionResponse } from './services/auth';
 import type {
+  AwsOidcPolicyPreset,
   PluginKind,
   Regions,
-  AwsOidcPolicyPreset,
 } from './services/integrations';
-import type { ParticipantMode } from 'teleport/services/session';
-import type { YamlSupportedResourceKind } from './services/yaml/types';
 import type { KubeResourceKind } from './services/kube/types';
+import type { YamlSupportedResourceKind } from './services/yaml/types';
```

Comment:
Looks like your editor rearranged the imports.

Reply:
Yeah, I ran a "sort imports" - is there a problem with putting the Y after K?

Reply:
No, I only wasn't sure if this was intentional :)